src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.navigation-list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: stretch;
+  height: 60px;
+  list-style: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to use a class for styling the list to avoid styling all lists in the document. Consider using a class like `.nav__list`.